docs/faqs.md

```diff
@@ -334,3 +337,10 @@ To isolate from this changes there are different strategies you can follow:
    and lockfiles. 
 
 Keep reading in the [consuming recipes section](consuming_recipes.md).
+
+## Why are version ranges not allowed?
+
+Version ranges are a useful Conan feature, find the documentation [here](https://docs.conan.io/en/latest/versioning/version_ranges.html). However in the context of ConanCenter they pose a few key challenges, most notably "Build Reproducibility".
+
+If consumers tries to download and build the recipe at a later time, it may resolve to a different package version that may not be compatable. In order to 
+prevent these types of issues we have decided to only allow exact requirements versions.
```

Comment:
```suggestion
If consumers try to download and build the recipe at a later time, it may resolve to a different package version that may not be compatible. In order to prevent these types of issues, we have decided to only allow exact requirements versions.
```